clay/amphora-storage-postgres
# Change node-redis for ioredis

# Changes

- Remove `redis` package
- Add `ioredis`package

# Testing
To test this change 

- Checkout this branch
- Create a link to this package
`npm link`
- In the project you want to test this change
`npm link [PATH_TO_AMPHORA_STORAGE_POSTGRES_PROJECT]`

Everything should work as expected.
